setup.py

```diff
@@ -49,6 +49,7 @@ def find_libs(src):
                               "lib/*.pk8",
                               "lib/weasel/armeabi/w",
                               "server/web_root/*" ] },
-  scripts = ["bin/drozer", "bin/drozer-complete"],
-  install_requires = ["protobuf==2.6.1","pyopenssl==16.2", "pyyaml==3.11"],
+  scripts = ["bin/drozer", "bin/drozer-complete", "bin/drozer-repository"],
+  install_requires = ["protobuf>=2.6.1","pyopenssl>=16.2", "pyyaml>=3.11"],
+  data_files = [('/etc/bash_completion.d',['scripts/drozer'])],
```

Comment:
I think we should support a minimum version of these dependencies instead of a specific version. Thoughts?